src/templates/en/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+      </p>
+
+      <p>
+        Mobile websites are run from within a mobile Chrome environment on an emulated Moto G4 device with a network speed equivalent to a 3G connection. Note that the emulated mobile User Agent self-identifies as Chrome 65 but is actually Chrome 84 under the hood.
+      </p>
+
+      <p>
+        There are two locations from which tests are run: California and Oregon USA. HTTP Archive maintains its own test agent hardware located in the <a href="https://archive.org">Internet Archive</a> datacenter in California. Additional test agents in <a href="https://cloud.google.com/compute/docs/regions-zones/#locations">Google Cloud Platform</a>'s us-west-1 location in Oregon are added as needed.
+      </p>
+
+      <p>
+        HTTP Archive's private instance of WebPageTest is kept in sync with the latest public version and augmented with <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a>. These are snippets of JavaScript that are evaluated on each website at the end of the test. The <a href="https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js">almanac.js</a> custom metric includes several metrics that were otherwise infeasible to calculate, for example those that depend on  DOM state.
+      </p>
+
+      <p>
+        The results of each test are made available as a <a href="https://en.wikipedia.org/wiki/HAR_(file_format)">HAR file</a>, a JSON-formatted archive file containing metadata about the web page.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="lighthouse"><a href="#lighthouse" class="anchor-link">Lighthouse</a></h3>
+
+      <p>
+        <a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a> is an automated website quality assurance tool built by Google. It audits web pages to make sure they don't include user experience antipatterns like unoptimized images and inaccessible content.
+      </p>
+
+      <p>
+        HTTP Archive runs the latest version of Lighthouse for all of its mobile web pages — desktop pages are not included because of limited resources. As of the August 2020 crawl, HTTP Archive used the <a href="https://github.com/GoogleChrome/lighthouse/releases/tag/v6.2.0">6.2.0</a> version of Lighthouse.
+      </p>
+
+      <p>
+        Lighthouse is run as its own distinct test from within <a href="#webpagetest">WebPageTest</a>, but it has its own configuration profile:
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+            <th>Config</th>
+            <th>Value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>CPU slowdown</td>
+            <td>1x/4x</td>
+          </tr>
+          <tr>
+            <td>Download throughput</td>
+            <td>1.475 Mbps</td>
```

Comment:
1.6Mbps

Reply:
Downloading one of the HARs shows this in the Lighthouse section:

```
                "downloadThroughputKbps": 1474.5600000000002,
```

Reply:
Lighthouse is reporting the measured thruput which is less than the connection speed (TCP overhead, etc).  The connection itself is 1.6Mbps.